examples/opf/experiments/anomaly/temporal/simple/description.py

```diff
@@ -195,8 +195,6 @@
             'synPermActiveInc': 0.1,
 
             'synPermInactiveDec': 0.01,
-
-            'randomSP': 1
```

Comment:
> How did you make this experiment specify learn=False to SP compute?

Actually I didn't. Look the discussion in this PR, specially the Subutai comment:

https://github.com/numenta/nupic/pull/1851

From I understood, as OPF tests pass `learn` parameter to `compute()`, then set randomSP is pointless..